src/Configuration/UserFunction.php

```diff
@@ -19,6 +19,14 @@ class UserFunction
      */
     public static function build($functions, array $params = [])
     {
+        if (!is_object($functions) && !is_array($functions)) {
```

Comment:
Aby to nepadalo, ak uzivatel zada do konfiguracie string. Pouziva sa to na viacerych miestach, preto takato univerzalna hlaska, aj s vypisanim zadanej hodnoty.